pytest/tests/mocknet/load_test_betanet.py

```diff
@@ -275,10 +48,7 @@ def get_test_accounts_from_args(argv):
                                        base_block_hash,
                                        rpc_infos=rpc_infos)
 
-    if need_deploy:
-        init_ft(node_account)
-
```

Comment:
just sanity checking that removing this was intentional

Reply:
yes, moved to `main()`